File: 10 Co wyświetli podany kod/treść.cpp
#include <iostream>
#include <vector>

using namespace std;

int main()
{
    vector<int> a = {9, 8, 7, 6, 5, 4, 3, 7};
    vector<int> b = {2, 2, 2, 2, 2, 2, 2, 1};
    vector<int> c = {5, 3, 1, 6, 8, 1, 6, 7};
    vector<vector<int>> abc = {a, b, c};

    for(auto x : abc)
    {
        switch(x[3])
        {
            case 2:
                cout << 2;
            case 4:
                cout << 4;
            case 6:
                cout << 6;
            default:
                cout << "<-_->";
        }
    }
    cout << abc[2][2];

    return 0;
}

File: 10 Co wyświetli podany kod/treść.py
a = [9, 8, 7, 6, 5, 4, 3, 7]
b = [2, 2, 2, 2, 2, 2, 2, 1]
c = [5, 3, 1, 6, 8, 1, 6, 7]
abc = [a, b, c]

for x in abc:
    v = x[3]
    if v <= 2:
        print(2, end="")
    if v <= 4:
        print(4, end="")
    if v <= 6:
        print(6, end="")
    print("<-_->", end="")

print(abc[2][2], end="")